### ISSUE TITLE
Disable mic button

### ISSUE THREAD

**Issue by sorryusernameisalreadytaken** [yuliskov/LeanKeyKeyboard#20]
THX for this great non-google kbd.

I did not use this kbd on my shieldtv with google services so the mic button does nothing. Is there a chance to disable it if needed or when the app does detect that there are no services for that?

**Comment by yuliskov** [comment]
Hi. As I know, there is no robust way to detect the mic. There is some
workarounds though. I could detect gapps presence for example. Pls tell me
does your box have google services installed.

Regards, Yuriy

**Comment by sorryusernameisalreadytaken** [comment]
yes, installed at the moment, but later I will change to LineageOS 6 on my nvidia shield tv without installing google-services but micro-G.

**Comment by yuliskov** [comment]
Seems that Micro-G services support voice search too. So it's not wise to
disable the mic for them.

Regards, Yuriy

**Comment by sorryusernameisalreadytaken** [comment]
I will never use the mic button but I like your kbd. Is there no chance to disable by long-click for example?

**Comment by yuliskov** [comment]
Maybe I'll add setting option for that.
Which key do you want to use instead?
I can't just remove it.

Regards, Yuriy

**Comment by anubis-genix** [comment]
Hello Yuriy,
your keyboard runs fine on my FireTV Stick 4K, but unfortunately the mic button crashes the keyboard and the app it was last used in. I can barely see a red mic symbol in a circle for a split second and then I'm left with a dark grey background with a lagging keyboard or no keyboard at all. Since you need to replace the button with another, how about putting keyboard settings in it's place? That would be great! 
Thanks once again for this wonderful app

**Comment by c0bnut** [comment]
Mic doesnt work on my FireStick Lite also, the mic is in the remote. If it could work with Alexa that would be fantastic.
